.devcontainer/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG VARIANT=16-bullseye
```

Comment:
Can the `VARIANT` be based on the [`.nvmrc`](https://github.com/ericcornelissen/git-tag-annotation-action/blob/8c5e3c0eed236768a199ee0d04becd00b82606e0/.nvmrc) to avoid repetition of the Node.js version for development.

Related to the following line from `.devcontainer.json` as well:

https://github.com/ericcornelissen/git-tag-annotation-action/blob/8c5e3c0eed236768a199ee0d04becd00b82606e0/.devcontainer/devcontainer.json#L6

Reply:
Based on what I was able to find online and try locally, it does not seem to be (easily) possible to load the `.nvmrc` value for the Docker base image.